src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -409,13 +409,18 @@ export default {
 	}
 
 	.pre-footer {
-		margin: 2rem 0;
+		margin-top: 2rem;
+		margin-bottom: 2rem;
```

Comment:
@ksekhri this is one you found the other day. I didn't realize I had overridden the margin of the default .row styles...